### ISSUE TITLE
Online demo link points to "strange" webpage

### ISSUE THREAD

**Issue by AndersBillLinden** [uxsolutions/bootstrap-datepicker#2643]
### Expected behaviour
When I click on "online demo", I want to come to a page where are use cases are listed.

### Actual behaviour
I am coming to the "bootstrap-datepicker sandbox" that is just a number of textboxes and checkboxes, all without explanation.

**Comment by fellwell5** [comment]
This is actually a demo of the bootstrap-datepicker.

You can test and customize your component.

**Comment by MrGrabazu** [comment]
I agree with @AndersBillLinden the page is strange, I want to test multidate, so I add some string date in the field but it doesn't seems to do anything, the datepicker doesn't change. 
![image](https://user-images.githubusercontent.com/6928672/151574548-208f337f-6c56-43b1-bf8f-036e63497d7c.png)

When I change format to a custom one I have an error in console and the picker disappear.
![image](https://user-images.githubusercontent.com/6928672/151574928-e02905a0-7cab-48a5-85dd-7a3d9089f8d9.png)
![image](https://user-images.githubusercontent.com/6928672/151574983-fe56fcb5-f8e8-4d28-b504-aabf992dfac8.png)

Maybe I'm doing something wrong, but I don't understand what.
